guidelines/1.3.5.md

```diff
@@ -106,6 +107,65 @@ android:labelFor="@id/edit_text"
 
 ---
 
+## Guidance for Flutter
+
+For widgets `Icon()` and `Text()` you can use:
+```dart 
+semanticLabel:'lable'
+```
+
+For `Image.asset()` you can use both:
+```dart
+semanticLabel:'lable'
+
+//and
+
+excludeFromSemantics:bool
+```
+
+For `TextFields()` the `InputDecoration()` will handle the semantics:
+
+```dart
+TextField(
+    decoration: InputDecoration(
+        hintText: 'Testing hint', // this has its own semantics
+        labelText: labelText.toUpperCase(), // this has its own semantics
+    ),
+)
+// You can achive more control using the different semantics widgets shown bellow
+```
+
+For widgets that need semantics but do not have them or
+for greater control of widget semantics you can use:
+
+```dart
+// Note that all the properties of these widgets are optional 
+
+// This widget will add semantics properties to its child though some of them might be overwriten if the child widget has its own semantic properties
+Semantics( //The `Semantics()` widget has many properties not all are listed here
+    lable:'label',
+    excludeSemantics: bool, // false by default
+    child: MyWidget()
+)
+
+// This will exclude the semantics of its child based on the bool value of `excluding:` which is true by default 
+ExcludeSemantics(
+    excluding:bool, // true by default
+    child: MyWidget(),
+)
+
+// This will merge all of the semantics of its child and its children all the way down the widget tree
+MergeSemantics(child: MyWidget())
+
+// This will block any semantics of a widget drawn bellow it e.g an alert should usually disallow interaction with any widget located "behind" the alert (even when they are still partially visible)
```

Comment:
typo: `below` and maybe change it to say: `an alert should usually block interaction` or `an alert should usually not allow interaction`